06-building-a-program-start-to-finish/setting-up-software-development-environments-(deployment)/Output/README.md

```diff
@@ -0,0 +1,8 @@
+#### Output 
+- [Link to review app](https://ddiei-inverted-index-sta-pr-14.herokuapp.com/)
+- [Link to Staging app](https://ddiei-inverted-index-staging.herokuapp.com/)
+- [link to production app](https://ddiei-inverted-index-app.herokuapp.com/)
```

Comment:
Your production app seems to be broken. I can't test that the application works, kindly fix and revert ASAP.

Also, your elements are out of alignment.
<img width="232" alt="screen shot 2017-03-15 at 9 10 49 am" src="https://cloud.githubusercontent.com/assets/929803/23939064/584bacfe-095f-11e7-8d59-cb5f62e49b2d.png">